cardano-sl.cabal

```diff
@@ -1077,29 +1077,31 @@ executable cardano-keygen
 executable cardano-launcher
   hs-source-dirs:      src/launcher
   main-is:             Main.hs
+  other-modules:       Launcher
   build-depends:       async
                      , base
                      , optparse-applicative
                      , cardano-report-server >= 0.1.0
                      , log-warper >= 0.4.3
                      , cardano-sl
+                     , bytestring
                      , directory
+                     , filepath
                      , process
                      , system-filepath
                      , text
                      , turtle
                      , universum
                      , wreq
                      , lens
+                     , yaml
   default-language:    Haskell2010
   ghc-options:         -threaded
                        -Wall
                        -O2
   default-extensions:  OverloadedStrings
                        RecordWildCards
                        TupleSections
-  if os(windows)
```

Comment:
Yes, I confirmed the cmd window pops up when running `cardano-launcher.exe` from Explorer. I'll add this back.